src/nav/AppNavigator.js

```diff
@@ -66,6 +67,7 @@ export default StackNavigator(
     notifications: { screen: NotificationsScreen },
     'welcome-help': { screen: WelcomeHelpScreen },
     welcome: { screen: WelcomeScreen },
+    'emoji-picker': { screen: EmojiPickerScreen },
```

Comment:
Just a tip: Try to maintain alphabetical order. Which helps a lot in searching when list becomes too large.